### ISSUE TITLE
No longer able to see LoadBalancer's status and floating IP with kubectl

### ISSUE THREAD

**Issue by oz123** [kubernetes/cloud-provider-openstack#812]
> /kind bug

**What happened**:
Earlier versions of cloud-provider showed the floating IP of a Loadbalancer. I could do somehting like this:
```
kubectl get service external-http-nginx-service --kubeconfig=${KUBECONFIG} -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
``` 
To get the floating IP of the LB. 

Now the status is completely empty.
```
$ kubectl get svc external-http-nginx-service -o json
{
    "apiVersion": "v1",
    "kind": "Service",
    "metadata": {
        "annotations": {
            "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"kind\":\"Service\",\"metadata\":{\"annotations\":{\"loadbalancer.openstack.org/floating-network-id\":\"c019250b-aea8-497e-9b3b-fd94020684b6\",\"service.beta.kubernetes.io/openstack-internal-load-balancer\":\"false\"},\"name\":\"external-http-nginx-service\",\"namespace\":\"default\"},\"spec\":{\"ports\":[{\"name\":\"http\",\"port\":80,\"targetPort\":80}],\"selector\":{\"app\":\"nginx\"},\"type\":\"LoadBalancer\"}}\n",
            "loadbalancer.openstack.org/floating-network-id": "c019250b-aea8-497e-9b3b-fd94020684b6",
            "service.beta.kubernetes.io/openstack-internal-load-balancer": "false"
        },
        "creationTimestamp": "2019-10-23T23:08:15Z",
        "name": "external-http-nginx-service",
        "namespace": "default",
        "resourceVersion": "2736",
        "selfLink": "/api/v1/namespaces/default/services/external-http-nginx-service",
        "uid": "fe97cb21-f5e9-11e9-b5e9-fa163e3658c6"
    },
    "spec": {
        "clusterIP": "10.100.115.31",
        "externalTrafficPolicy": "Cluster",
        "ports": [
            {
                "name": "http",
                "nodePort": 31291,
                "port": 80,
                "protocol": "TCP",
                "targetPort": 80
            }
        ],
        "selector": {
            "app": "nginx"
        },
        "sessionAffinity": "None",
        "type": "LoadBalancer"
    },
    "status": {
        "loadBalancer": {}
    }
}
```

I know the loadbalancer is alive and ok from openstack side of things:

```
openstack floating ip list -c "Fixed IP Address" -c "Floating IP Address" -f value | grep $(openstack loadbalancer show kube_service_kubernetes_default_external-http-nginx-service -f value -c vip_address) | cut -d" " -f 1
```
- openstack-cloud-controller-manager version: 1.14
- OS (e.g. from /etc/os-release): ubuntu 16.04
- Install tools: kubeadm

Is this behavior desired?

**Comment by kayrus** [comment]
@oz123 do you want to tell that openstack-cloud-controller prior to 1.14 showed LB FIP, but since 1.14 it stopped?

**Comment by oz123** [comment]
@kayrus yes indeed.

**Comment by kayrus** [comment]
@oz123 have you tried the latest version? There was an issue with the LB name detection fixed in #742

**Comment by oz123** [comment]
Thanks for the hint. I'm using k8s 1.14 so I thought I should match the ccm version. I will try  the latest version and report back.
